src/Websites/Websites.Test/ScenarioTests/WebsitesTestRunner.cs

```diff
@@ -21,12 +35,11 @@ protected WebsitesTestRunner(ITestOutputHelper output)
                     @"../AzureRM.Storage.ps1"
```

Comment:
There is a Common.ps1 file under the ScenarioTests folder. Please load it in the common ps script section as it was there in the controller class.